loader/loader.go

```diff
@@ -479,18 +482,37 @@ func (l *Loader) protoEnum(tspec *ast.TypeSpec) (*desc.EnumDescriptorProto, erro
 	return enum, nil
 }
 
-func (l *Loader) protoType(expr ast.Expr, pkg *types.Package) (desc.FieldDescriptorProto_Type, string) {
+func (l *Loader) protoType(expr ast.Expr, pkg *types.Package) (desc.FieldDescriptorProto_Type, desc.FieldDescriptorProto_Label, string) {
 	if pkg == nil {
 		pkg = l.tpkg
 	}
 	switch x := expr.(type) {
 	case *ast.Ident:
 		switch x.Name {
 		case "string":
-			return desc.FieldDescriptorProto_TYPE_STRING, x.Name
+			return desc.FieldDescriptorProto_TYPE_STRING, 0, x.Name
+		case "int", "int32":
+			return desc.FieldDescriptorProto_TYPE_INT32, 0, x.Name
```

Comment:
This seems fine - it can be a pain to work with `int32` or `int64` in Go code. But I wonder if it would be better to translate `int` as `INT64`. Then again, I guess that if anyone really needs the bits, they should use a 64-bit number for portability.